Update resources for Chef Infra Client 15.5 (#2195)
chef/chef-web-docs: chef_master/source/resource_chef_sleep.rst

=====================================================
chef_sleep resource
=====================================================
`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/resource_chef_sleep.rst>`__

> Use the **chef_sleep** resource to set the number of seconds to sleep during a Chef Infra Client run. Only use this resource when a command or service exits successfully but is not ready for the next step of the recipe.

Reviewer: Is it fair to say that it will pause an Infra Client run?

```suggestion
The **chef_sleep** resource will pause a Chef Infra Client run for a set number of seconds. Only use this resource when a command or service exits successfully but is not ready for the next step of the recipe.
```

**New in Chef Infra Client 15.5.**

Syntax
=====================================================

The chef_sleep resource has the following syntax:

.. code-block:: ruby

  chef_sleep 'name' do
    seconds      String, Integer # default value: 'name' unless specified
    action       Symbol # defaults to :sleep if not specified
  end

where:

* ``chef_sleep`` is the resource.
* ``name`` is the name given to the resource block.
* ``action`` identifies which steps Chef Infra Client will take to bring the node into the desired state.
* ``seconds`` is the property available to this resource.

Actions
=====================================================

The chef_sleep resource has the following actions:

``:sleep``

>    Sleep the Chef Infra Client run for a specified number of seconds

Reviewer: ```suggestion
   Pause the Chef Infra Client run for a specified number of seconds.
```
